Python/README.md

```diff
@@ -84,6 +97,8 @@
 - [Labelling_Binarizer_Instance](/Python/Labelling_Binarizer_Instance)
 - [Large_File_Identifier](/Python/Large_File_Identifier)
 - [Launching_Programs_from_Python](/Python/Launching_Programs_from_Python)
+- [Linkedin Post Scrapper](/Python/Linkedin Post Scrapper)
```

Comment:
URL with Spaces in not accepted, this would look like this `/Python/LinkedIn%20Posts%20Scrapper`

A better solution would be to Rename the Directory itself.

So what I want is, please rename the directory to `Linkedin Post Scrapper` then do the changes accordingly.

Would it be possible?

Reply:
Sure. I will do it